hardhat-example/scripts/transferHbars.js

```diff
@@ -0,0 +1,14 @@
+const hre = require('hardhat');
+
+module.exports = async () => {
+  const provider = new hre.ethers.providers.JsonRpcProvider('http://localhost:7546');
+  const wallet = new hre.ethers.Wallet(process.env.OPERATOR_PRIVATE_KEY, provider);
+  const walletReceiver = new hre.ethers.Wallet(process.env.RECEIVER_PRIVATE_KEY, provider);
+
+  console.log(`Balance before tx: ${await walletReceiver.getBalance()}`);
+  await wallet.sendTransaction({
+    to: walletReceiver.address,
+    value: 10000000000
```

Comment:
nit: add comment for amount of value with unit e.g.
```suggestion
    value: 10000000000 // 100 hbar
```

Reply:
TINYBAR to WEIBAR coefficient is 10000000000, so that's 1 tinybar. I bump it to 10 tinybars.